firmware/Core/Src/unit_tests/test_telecommand_parser.c

```diff
@@ -122,3 +122,16 @@ uint8_t TEST_EXEC__TCMD_ascii_to_double() {
 
     return 0;
 }
+
+uint8_t TEST_EXEC_TCMD_TCMD_parse_full_telecommand() {
+    TCMD_parsed_tcmd_to_execute_t parsed_tcmd;
+    TEST_ASSERT_TRUE(TCMD_parse_full_telecommand("CTS1+hello_world()!", TCMD_TelecommandChannel_DEBUG_UART, &parsed_tcmd) == 0);
```

Comment:
In this test case, please validate that the members of the struct are set correctly as well, as best as you can.